core/src/commonMain/kotlin/com/powersync/utils/Strings.kt

```diff
@@ -0,0 +1,7 @@
+package com.powersync.utils
+
+internal object Strings {
+    fun quoteIdentifier(s: String): String {
```

Comment:
This can quote any string, not just identifiers, please rename this to `wrapInQuotes` or something similar. 

Also, instead of `internal object` consider extending `String`, like 

```
fun String.wrapInQuotes(): String {
    return if (this.startsWith("\"") && this.endsWith("\"")) {
        this
    } else {
        "\"$this\""
    }
}
```

Allowing it to be used like this:

```
var a = "foo"
var b = a.wrapInQuotes()
println(b) // Outputs: "foo"
```

Reply:
The quoting rules used for this are specific to SQLite identifiers - text literals use single quotes.

When extending String, will it still only available inside the library?